.github/workflows/promptflow-evals-local-e2e-test.yml

```diff
@@ -0,0 +1,95 @@
+name: promptflow-evals-local-e2e-test
```

Comment:
Can we rename the file to "promptflow-evals-e2e-test-local.yaml"?

And rename the existing "promptflow-evals-e2e-test.yaml" to "promptflow-evals-e2e-test-azure.yaml"? This will make it clearer how the tests are organized.

Also, can we configure "promptflow-evals-e2e-test-azure.yaml" to only run non-local tests to avoid running local tests twice in two pipelines?

Reply:
Fixed.